File: BOJ11720/oognuyh.java
package BOJ11720;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

class oognuyh {

  public static void main(String[] args) throws IOException {
    try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in))) {
      bufferedReader.readLine().trim();

      System.out.println(
          Arrays.stream(bufferedReader.readLine().trim().split(""))
              .mapToInt(Integer::parseInt)
              .sum());
    }
  }
}

File: BOJ2480/oognuyh.kt
import java.io.BufferedReader
import java.io.InputStream
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) { 
    val numbers = readLine().split(" ").map { it.toInt() }.toIntArray()
    val (a, b, c) = numbers

    println(
        when {
            a == b && b == c -> 10000 + a * 1000
            a == b || a == c || b == c -> 1000 + (if (b == c) b else a) * 100
            else -> numbers.maxOrNull()!! * 100
        }
    )
}